cg/cli/workflow/balsamic/options.py

```diff
@@ -5,6 +5,14 @@
 OPTION_DRY = click.option(
     "-d", "--dry-run", help="Print command to console without executing", is_flag=True
 )
+OPTION_GENOME_VERSION = click.option(
+    "-g",
```

Comment:
This one might not add much - I would go with only a long version of this command